wandb/sweeps/test_bayes_search.py

```diff
@@ -266,11 +266,27 @@ def test_runs_bayes_nan():
             'value': 5
         }
     }, {'loss': 'NaN'}, [])
+    r5 = Run('b', 'finished', {
+        'v1': {
+            'value': 9
+        },
+        'v2': {
+            'value': 10
+        }
+    }, {'loss': 'inf'}, [])
+    r6 = Run('b', 'finished', {
+        'v1': {
+            'value': 9
+        },
+        'v2': {
+            'value': 10
+        }
+    }, {'loss': float('inf')}, [])
     # need two (non running) runs before we get a new set of parameters
-    runs = [r1, r2, r3, r4]
+    runs = [r1, r2, r3, r4, r5, r6]
     sweep = {'config': sweep_config_2params, 'runs': runs}
     params, info = bs.next_run(sweep)
-    assert params['v1']['value'] == 10 and params['v2']['value'] == 2
+    assert params['v1']['value'] == 6 and params['v2']['value'] == 6
```

Comment:
we are currently having some issues with this test (returning 10,1 instead of 10,2 as asserted in the old side of the diff). for the sake of monitoring that test, can we break this out into a separate test and keep the old one as is.